src/helpers/__tests__/retry.test.ts

```diff
@@ -0,0 +1,99 @@
+import {AxiosPromise} from 'axios'
+import {retryRequest} from '../retry'
+
+describe('retry', () => {
+  const retryCallback = jest.fn()
+  const createResultWithErrors = (errors: any[]): (() => AxiosPromise) => {
+    let i = -1
+
+    return () => {
+      i = i + 1
+      if (errors[i] === undefined) {
+        return Promise.resolve({
+          config: {},
+          data: {},
+          headers: undefined,
+          status: 200,
+          statusText: '',
+        })
```

Comment:
are you ever reading the resolved value? We could just 
```suggestion
        return Promise.resolve()
```

Reply:
It will complain that it is not the correct type.